### PR TITLE
Features/37662 search filter table component

### PR DESCRIPTION
**Pull request by HiirenP**

### CODE REVIEW

File: src/components/Table/Table.tsx [hunk 1]
```diff
@@ -98,6 +95,7 @@ class Table extends React.Component<Props, State> {
   }
 
   componentWillReceiveProps(newProps: Props) {
+    debugger;
     if (newProps.filterData) {
```
Comment (ankitsaraogi) [inline src/components/Table/Table.tsx]: Remove debugger from here.

Comment (ankitsaraogi) [inline src/components/Table/Table.tsx]: Remove debugger from here.

**Comment by HiirenP** [comment]
@ankitsaraogi  Resolved!!

**Comment by ankitsaraogi** [comment]
@HiirenP : Its again showing conflict, sorry my mistake I forgot to merge it last time. Can you please resolve it, I will merge it right away.

**Comment by HiirenP** [comment]
@ankitsaraogi conflicts have been resolved!

**Pull request merged**